src/types/Enums.ts

```diff
@@ -0,0 +1,2 @@
+export const picProfileColMob = 3
```

Comment:
Check out plz - what enums mean in ts

Reply:
Refactored the Enums.ts file and the use of the number of columns value.